File: code-samples/python/feeds@products@update.py
import requests

url = "https://vendor-api-staging.jumia.com/feeds/products/update"

payload = {
    "shopId": "string",
    "products": [
        {
            "id": "string",
            "name": {
                "value": "string",
                "translations": [
                    {
                        "language": "string",
                        "value": "string"
                    }
                ]
            },
            "description": {
                "value": "string",
                "translations": [
                    {
                        "language": "string",
                        "value": "string"
                    }
                ]
            },
            "parentSku": "string",
            "sellerSku": "string",
            "barcodeEan": "string",
            "brand": {
                "code": 0,
                "name": "string"
            },
            "category": {
                "code": 0,
                "name": "string"
            },
            "additionalCategories": [
                {
                    "code": 0,
                    "name": "string"
                }
            ],
            "images": [
                {
                    "url": "string",
                    "primary": True
                }
            ],
            "price": {
                "value": 0,
                "salePrice": {
                    "value": 0,
                    "startAt": "string",
                    "endAt": "string"
                }
            },
            "stock": 0,
            "attributes": [
                {
                    "name": "string",
                    "value": "string",
                    "translations": [
                        {
                            "language": "string",
                            "value": "string"
                        }
                    ]
                }
            ]
        }
    ]
}
headers = {"Authorization": "Bearer "}

response = requests.request("POST", url, json=payload, headers=headers)

print(response.text)

File: code-samples/go/feeds@products@update.go
package main

import (
  "fmt"
  "strings"
  "net/http"
  "io/ioutil"
)

func main() {

  url := "https://vendor-api-staging.jumia.com/feeds/products/update"

  payload := strings.NewReader("{\n  \"shopId\": \"string\",\n  \"products\": [\n    {\n      \"id\": \"string\",\n      \"name\": {\n        \"value\": \"string\",\n        \"translations\": [\n          {\n            \"language\": \"string\",\n            \"value\": \"string\"\n          }\n        ]\n      },\n      \"description\": {\n        \"value\": \"string\",\n        \"translations\": [\n          {\n            \"language\": \"string\",\n            \"value\": \"string\"\n          }\n        ]\n      },\n      \"parentSku\": \"string\",\n      \"sellerSku\": \"string\",\n      \"barcodeEan\": \"string\",\n      \"brand\": {\n        \"code\": 0,\n        \"name\": \"string\"\n      },\n      \"category\": {\n        \"code\": 0,\n        \"name\": \"string\"\n      },\n      \"additionalCategories\": [\n        {\n          \"code\": 0,\n          \"name\": \"string\"\n        }\n      ],\n      \"images\": [\n        {\n          \"url\": \"string\",\n          \"primary\": true\n        }\n      ],\n      \"price\": {\n        \"value\": 0,\n        \"salePrice\": {\n          \"value\": 0,\n          \"startAt\": \"string\",\n          \"endAt\": \"string\"\n        }\n      },\n      \"stock\": 0,\n      \"attributes\": [\n        {\n          \"name\": \"string\",\n          \"value\": \"string\",\n          \"translations\": [\n            {\n              \"language\": \"string\",\n              \"value\": \"string\"\n            }\n          ]\n        }\n      ]\n    }\n  ]\n}")

  req, _ := http.NewRequest("POST", url, payload)

  req.Header.Add("Authorization", "Bearer ")

  res, _ := http.DefaultClient.Do(req)

  defer res.Body.Close()
  body, _ := ioutil.ReadAll(res.Body)

  fmt.Println(res)
  fmt.Println(string(body))

}